debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySQLCube.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+import com.github.dockerjava.api.DockerClient;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Configuration.Builder;
+
+enum MySQLCube {
```

Comment:
This could also use JavaDoc to help other devs understand what each literal means and how the enum is used.